com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs

```diff
@@ -128,6 +129,20 @@ public override void GetActiveBlocks(ref TargetActiveBlockContext context)
             context.AddBlock(BlockFields.SurfaceDescription.CoatSmoothness,     clearCoat);
         }
 
+        public override void CollectShaderProperties(PropertyCollector collector, GenerationMode generationMode)
+        {
+            base.CollectShaderProperties(collector, generationMode);
+            collector.AddShaderProperty(new ColorShaderProperty()
+            {
+                overrideReferenceName = "_EmissionColor",
+                hidden = true,
+                overrideHLSLDeclaration = true,
+                hlslDeclarationOverride = HLSLDeclaration.UnityPerMaterial,
+                value = new Color(1.0f, 1.0f, 1.0f, 1.0f)
+            });
```

Comment:
What does the _EmissionColor property do?

Reply:
Aside from getting rid of the warning spew?  :)

Reply:
No idea.  It's not really in Shader Graph.  It's really a pipeline thing (even though the material editor GUI is in SG).

Reply:
So, there is a custom ShaderGUI registered by the URP Lit SubTarget which "asserts" this property on the material editor.  If it's not there, we get the error.  It's always declared by the hand-written Lit.shader.  This adds it in the same fashion. The difference is the default value, which is the one thing I'm still not sure about (Lit.shader has it as a 3-component 0).

Reply:
Right, but is it the correct thing to add this property, or remove the warning?   Need to find someone who can answer that question..  :)

Reply:
ah, nevermind, I read the bug -- Kasper says to do it, he should know..   :)

Reply:
he should know the correct default as well.

Reply:
@KEngelstoft mind enlightening us as to what this value does?   :)

Reply:
Yeah, I'd put him on this PR as reviewer for precisely that reason :-)

Reply:
ok, so it's kind of equivalent to MainTex/MainColor -- a fixed function definition of a GI input.
MainTex/MainColor define the bounce color for GI,
and _EmissionMap/_EmissionColor define the emission color for GI.

Reply:
In the built in Standard shader, there is a checkbox for Emission. Once checked, the emission HDR color property is shown and the default color is black.
There is an enum dropdown letting the user choose if emission should control realtime, baked or no GI.
![image](https://user-images.githubusercontent.com/6929868/113843662-1056a580-9794-11eb-954f-f9c0af115975.png)